opencti-platform/opencti-front/src/utils/defaultRepresentatives.ts

```diff
@@ -92,6 +92,9 @@ export const defaultKey = (n: any) => {
 // equivalent to querying representative.main
 export const getMainRepresentative = (n: any, fallback = 'Unknown') => {
   if (!n) return '';
+  if (n.name === 'Unknown') {
```

Comment:
I don't get this change

Reply:
In this function, "Unknown" is indirectly handled as a fallback value. It will only be returned: if none of the properties or function calls preceding it in the mainValue assignment are truthy. 
To ensure that the name "Unknown" is returned if it is set as n.name, we could modify the function to handle the "Unknown" case explicitly before.